src/View/SSViewer.php

```diff
@@ -86,6 +87,7 @@ class SSViewer implements Flushable
      *
      * @config
      * @var string
+     * @deprecated 5.4.0 Will be moved to SilverStripe\View\SSTemplateEngine.global_key
      */
     private static $global_key = '$CurrentReadingMode, $CurrentUser.ID';
```

Comment:
Whole bunch of stuff like this that's template-specific which is getting moved over to the new engine.